src/zkevm_specs/opcode/sload_sstore.py

```diff
@@ -0,0 +1,97 @@
+from typing import Sequence, Tuple
+from zkevm_specs.opcode.storage import Storage
+from zkevm_specs.opcode.stack import Stack
+from ..encoding import U8, U128, U256, is_circuit_code, u8s_to_u256
+
+OP_SLOAD = 0x54
+OP_SSTORE = 0x55
+
+G_WARM_ACCESS = 100
+G_COLD_SLOAD = 2100
+G_S_SET = 20000
+G_S_RESET = 2900
+
+R_S_CLEAR = 15000
+
+"""
+The Yellow Paper defines:
+We remind the reader that the checkpoint ("original") state sigma_0 is the
+state if the current transaction were to revert.
+Therefore to compute the required gas for an SSTORE we need both the current
+storage and the original storage from the beginning of the transaction.
+"""
+
+
+@is_circuit_code
+def check_storage_ops(
+    opcode: U8,
+    storage: Storage,
+    key8s: Sequence[U8],
+    new_value8s: Sequence[U8],
+    original_value8s: Sequence[U8],
+    current_value8s: Sequence[U8],
```

Comment:
you mention in the markdown spec that you use 1 word instead of 32bytes. (and you are right about that).

I think there's no need to convert `U256` to and from `[U8]`, which is redundant.